cpu/esp8266/espressif/README.md

```diff
@@ -0,0 +1 @@
+The files in this directory are either from the [ESP8266_NONOS_SDK](https://github.com/espressif/ESP8266_NONOS_SDK.git) or from the [ESP_RTOS_SDK](https://github.com/espressif/ESP8266_RTOS_SDK.git) for ESP8266. All of these files are copyright of Tensilica Inc. and Cadence Design Systems Inc., repectively. Please note the copyright notice in these files.
```

Comment:
I wonder whether it makes sense to move cpu/esp8266/espressif/ and cpu/esp8266/xtensa/ to cpu/esp8266/vendor?

Most files don't really adhere to our coding conventions. Also it would be nice to seperate the actual changes necessary for RIOT, both so we know what to review and so we can update when necessary.

Opinions?

Reply:
@kaspar030 
Placing ```cpu/esp8266/espressif/``` and ```cpu/esp8266/xtensa/``` to ```cpu/esp8266/vendor``` shouldn't be a problem.

Separating changes to be able to make update is quite difficult since every file was changed to satisfy RIOT's static tests and I don't see a necessity. The files in ```cpu/esp8266/xtensa``` were derived from FreeRTOS 9.0 which is not developed further. Thus, future updates are unlikely.

Please don't forget, all changes would require additional working time and I'm quite busy at the moment :worried:

Reply:
Yep - @kaspar030 can we focus on a straight path to convergence here?

Reply:
> Please don't forget, all changes would require additional working time and I'm quite busy at the moment

Alright, let's keep them where they are for now. We can move them next time they need to be updated.